cpssEnabler/linuxNoKernelModule/drivers/intDriver.c

```diff
@@ -273,8 +320,19 @@ static ssize_t mvIntDrv_write(struct file *f, const char *buf, size_t siz, loff_
 					   (unsigned)cmdBuf[3]));
 	if (pdev) {
 		int rc;
+
+		pr_info("%s: Got request to enable MSI for %x:%x:%x\n",
```

Comment:
The arguments to pr_info() don't look correct here.
Cut-and-paste error from line 363?

Reply:
Has no functional impact. Will be fixed on next PR.